Z_Master_Skills/Config/recipes.xml

```diff
@@ -39,18 +39,18 @@
 			<ingredient name="resourceForgedSteel" count="25"/>
 		</recipe>
 		
-		<!-- <recipe name="solarCell" count="1" craft_time="1500" craft_area="workbench" tags="learnable,perkMachineGunner"> -->
-			<!-- <ingredient name="resourceElectricPartsImp" count="10"/> -->
-			<!-- <ingredient name="resourceBrokenGlass" count="100"/> -->
-			<!-- <ingredient name="resourceScrapPolymers" count="50"/> -->
-			<!-- <ingredient name="resourceForgedSteel" count="20"/> -->
-			<!-- <effect_group> -->
-				<!-- <passive_effect name="CraftingIngredientCount" level="2,6" operation="base_add" value="3,15" tags="resourceElectricPartsImp"/> -->
-				<!-- <passive_effect name="CraftingIngredientCount" level="2,6" operation="base_add" value="50,250" tags="resourceBrokenGlass"/> -->
-				<!-- <passive_effect name="CraftingIngredientCount" level="2,6" operation="base_add" value="30,150" tags="resourceScrapPolymers"/> -->
-				<!-- <passive_effect name="CraftingIngredientCount" level="5,6" operation="base_add" value="10,50" tags="resourceForgedSteel"/> -->
-			<!-- </effect_group> -->
-		<!-- </recipe> -->
+		<recipe name="solarCell" count="1" craft_time="1500" craft_area="workbench" tags="learnable,perkMachineGunner">

```

Comment:
```suggestion
		<recipe name="solarCell" count="1" craft_time="1500" craft_area="workbench" tags="learnable,perkAdvancedEngineering">

```

@BlackRabbitMsk I didn't want to play with this because it's unrelated to the fix in this PR, but is `perkMachineGunner` correct here? Should this be `perkAdvancedEngineering` (I think?) instead? 

I'm still learning my way around these XMLs but I figured I'd mention it while this PR's open anyway. I added it as a suggestion so you can just click it to update the line if I'm correct.